ffi/framebuffer_mxcfb.lua

```diff
@@ -271,6 +306,20 @@ local function mxc_update(fb, ioc_cmd, ioc_data, is_flashing, waveform_mode, x,
         return
     end
 
+    -- Wake the EPDC up manually (don't ask me why this would make any sort of sense, it's simply what Nickel does on devices where we *can* do it...)
+    -- NOTE: Technically, it makes a little more sense when you put it in context of *how* Nickel does that: it's not actually tied to the refresh or the wait there,
+    --       but to *touch* input: on any registered *touch* input (physical buttons are exempt, for some reason), Nickel will forcibly awaken the EPDC,
+    --       unless that was already done less than roughly 1.5s ago.
+    --       Think of something like some properly implemented & tuned kernels with the interactive cpufreq governor that will boost on touch input, for instance.
+    --       In practice, that still trips *very* close to the ioctl (but may not always come *before* a [wait -> refresh -> wait] sandwich;
+    --       e.g., I mostly see it before the refresh on NXP & sunxi, but I mostly see it in front of everything on MTK).
+    --       TL;DR: Because, on devices flagged !hasReliableMxcWaitFor, we were seeing deadlock issues with refreshes that aren't necessarily tied to input,
+    --       (and for simplicity's sake), we simply unconditionally do this here as early possible.
+    -- NOTE: This might be a gigantic red herring, and simply a case of the very few extra cpu cycles involved throwing off the race...
```

Comment:
lol

Reply:
Wheee \o/

Reply:
On the upside, I genuinely haven't managed to hang the Clara 2E with jump markers since ;).

Reply:
The downside being I *never* actually ùanaged to hang it with anything else (specifically, Notifications).

My only other hang was *extremely* overenginereed, as it involved [ftrace](https://github.com/NiLuJe/FBInk/blob/master/utils/finger_trace.c), which will positively spam refresh ioctls ;).